DEI.md

```diff
@@ -0,0 +1,56 @@
+
+Scope of the DEI.md File
+
+Our project prioritizes and reflects on DEI through a regular review of project policies and performance. This reflection is documented in the following DEI.md file based on specific [CHAOSS project](https://chaoss.community) DEI metrics. 
+
+### [Project Access](https://chaoss.community/?p=4953)
+Project access is addressed in our project through various efforts. Through these efforts, we aim to support access for all. Specific efforts in our project include:
+* Our Django Fellows span global time zones and work asynchronously with contributors across the world.
+* The Django documentation and admin has multiple translations.
+* All areas of Django are considered open to community contributions. 
+* Django Software Foundation membership, which recognises contributors to Django, is open to non-code contributors, including contributions through events, blog posts, and comments in issues and merge requests.
+* We have an Accessibility Team dedicated to improving the accessibility of Django.
+* DjangoCons offer opportunity grants to make the conferences accessible to those who are new and/or financially constrained.
+* DjangoCons make it a priority to offer childcare reimbursements and private facilities for breastfeeding.
+* DjangoCon talks have human captioners which make talks more accessible - especially to those hard of hearing and language learners.
+
+_Note: the extra provisions and opportunity grants are only possible at DjangoCons due to our generous sponsors. 
+DjangoCons have diversity and accessibility packages which allow sponsors to specifically fund these efforts._
+  
+### [Communication Transparency](https://chaoss.community/?p=4957)
+Communication Transparency is addressed in our project through a variety of different efforts. Through these efforts, we aim to support transparency for all. Specific efforts in our project include: 
+* All project documentation is publicly available and [djangoproject.com](https://djangoproject.com).
+* Decisions that impact Django are shared via the Django Software Board [meeting minutes](https://www.djangoproject.com/foundation/minutes/).
+* [Statistics of Code of Conduct reports](https://github.com/django/code-of-conduct/blob/main/statistics.md) are publicly available.
+* DjangoCon talks are publicly available - [DjangoCon Europe](https://www.youtube.com/@djangoconeurope), [DjangoCon US](https://www.youtube.com/@DjangoConUS)
+           
+### [Newcomer Experiences](https://chaoss.community/?p=4891)
+The newcomer experience is addressed in our project through a variety of different efforts. Through these efforts, we aim to support the newcomer experience for all new members. Specific efforts in our project include:
+* A quick start guide for newcomers is available on our website and made available through the main website navigation.
+* Django has an automatic welcome message for first time contributors to Django who are awaiting a response to a merge request.
```

Comment:
Shall we also include the specific channels in Discourse and the forum for seeking help?
Also, Djangonauts space!!!